tailwind.config.js

```diff
@@ -3,6 +3,7 @@ export default {
   content: ['./index.html', './src/**/*.{vue,js,ts,jsx,tsx}'],
   theme: {
     screens: {
+      mb: '368px',
```

Comment:
```mb``` nghĩa là gì em nhỉ, đặt tên phải có nghĩa nhé
vd: msm -> mini smartphone
```suggestion
msm: 368px
```